Clinical-Genomics/scout
# Add link chanjo2 coverage report on gene panel page

This PR adds a functionality or fixes a bug.
- Closes #4512 and Other partial fix to #4000

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>


**How to test**:
1. Deploy on cg-vm1 and go to case [safeguinea](https://scout-stage.scilifelab.se/cust000/F0006358-mip7). This case has an individual with .d4 files associated
2. From the panel containing the gene panels, click on the link of a gene panel:

<img width="732" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/33e1835d-1d85-4d29-8bd1-c8daa4e7c1dd">

3. Now you should be able to see the link to the chanjo2 report on the gene panel page (absent on main branch):

<img width="1093" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/9deced19-0a54-4532-9c7b-00620331ba9f">

4. Click on the link and make sure the report is generated




**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
